compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Primitives.hs

```diff
@@ -6,14 +6,20 @@
 
 
 -- | The Daml-LF primitives, matched with their type, and using 'primitive' on the libraries side.
-module DA.Daml.LFConversion.Primitives(convertPrim) where
+module DA.Daml.LFConversion.Primitives(convertPrim, convertPrimVersioned) where
 
+import           DA.Daml.LFConversion.ConvertM
 import           DA.Daml.LF.Ast
 import           DA.Daml.UtilLF
 import           DA.Pretty (renderPretty)
 import qualified Data.Text as T
 import qualified Data.List as L
 
+convertPrimVersioned :: Version -> String -> Type -> ConvertM Expr
```

Comment:
Phew that took longer than expected - making conversion errors strict caused some issues because previously we were relying on `error` calls being unevaluated which we did checks with pretty errors in LFConversion.

Reply:
oh, that's interesting. Thank you for fixing it!